### PR TITLE
adding m-btc and solvBTC prices in prices_linea_tokens.sql

### PR DESCRIPTION
**Pull request by 0xroll**
adding m-btc and solvBTC on linea

### Contribution type
Please check the type of contribution this pull request is for:

- [x] Adding to existing spell lineage

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [adding m-btc and solvbtc in prices table]

---

### Additional information
Please provide any additional information that might help us review your pull request:

https://api.coinpaprika.com/v1/coins/m-btc-merlins-seal-btc
https://api.coinpaprika.com/v1/coins/solvbtc-solv-protocol-solvbtc

---

### CODE REVIEW

File: scripts/token_checker.py [hunk 1]
```diff
@@ -26,7 +26,8 @@ def __init__(self):
                             "zksync": "zksync-zksync",
                             "zora": "eth-ethereum",
                             "mantle": "mnt-mantle",
-                            "blast": "blast-blast"
+                            "blast": "blast-blast",
+                            "linea": "eth-ethereum"
```
Comment (Hosuke) [inline scripts/token_checker.py:30]: Adding linea support similar to zora.

(Currently linea and zora are still not in the coinpaprika list: https://api.coinpaprika.com/v1/contracts/)

File: models/prices/linea/prices_linea_tokens.sql [hunk 2]
```diff
@@ -24,4 +24,6 @@ FROM
     ,('uni-uniswap', 'linea', 'UNI', 0x636B22bC471c955A8DB60f28D4795066a8201fa3, 18)
     ,('link-chainlink', 'linea', 'LINK', 0x5B16228B94b68C7cE33AF2ACc5663eBdE4dCFA2d, 18)
     ,('dai-dai', 'linea', 'DAI', 0x4AF15ec2A0BD43Db75dd04E62FAA3B8EF36b00d5, 18)
+    ,('m-btc-merlins-seal-btc', 'linea', 'M-BTC',0xe4D584ae9b753e549cAE66200A6475d2f00705f7, 18)
```
Comment (Hosuke) [inline models/prices/linea/prices_linea_tokens.sql:27]: https://lineascan.build/token/0xe4D584ae9b753e549cAE66200A6475d2f00705f7

File: models/prices/linea/prices_linea_tokens.sql [hunk 3]
```diff
@@ -24,4 +24,6 @@ FROM
...
+    ,('solvbtc-solv-protocol-solvbtc', 'linea', 'solvBTC', 0x5FFcE65A40f6d3de5332766ffF6A28BF491C868c, 18)
```
Comment (Hosuke) [inline models/prices/linea/prices_linea_tokens.sql:28]: https://lineascan.build/token/0x5FFcE65A40f6d3de5332766ffF6A28BF491C868c

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @0xroll

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**